numba/byteflow.py

```diff
@@ -15,6 +16,43 @@
 _logger = logging.getLogger(__name__)
 
 
+_EXCEPT_STACK_OFFSET = 6
+_FINALLY_POP = _EXCEPT_STACK_OFFSET if PYVERSION >= (3, 8) else 1
+
+
+@total_ordering
+class BlockKind(object):
```

Comment:
Was enum not suitable?

Reply:
I needed stricter typing at comparison to prune out bugs. Tried enum but it was not strict enough.